README.md

```diff
@@ -2,49 +2,42 @@
 
 _A tool to find ways to get involved on campus._
 
-## Getting Started
+## Contributing
 
-This project requires a working [Node.js](https://nodejs.org/en/) installation
-(at least v14) with NPM, which comes bundled with Node.js (at least v8). (If you
-use Yarn, that's fine, too. Just beware of the commands.)
+We are always open to contributions to the project. If you would like to contribute,
+please join our [Discord](http://discord.utdnebula.com/) and ask for how to access the Jupiter Dev Team
 
-First, clone the repository.
+Please make sure you have at least NodeJS v18.0.0 or greater installed before continuing.
+
+### Getting Started
+
+Start by cloning the repository to your local machine.
 
 ```bash
-git checkout https://github.com/UTDNebula/jupiter.git
+git clone https://github.com/UTDNebula/jupiter.git
 ```
 
-Make sure to install the dependencies:
+Next, navigate to the project directory and install the dependencies.
 
 ```bash
+cd jupiter
 npm install
 ```
 
-Now you can run the development server:
+Make sure you have a `.env` file in the root of the project. If you do not, copy the `.env.example` file and rename it to `.env`. Request the `.env` file from the Jupiter Dev Team.
+
+Finally, start the development server.
 
 ```bash
 npm run dev
-# or
-yarn dev
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
-
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
-
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
-
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+### Branching
 
-## Learn More
+When working on a new feature, please create a new branch with the following naming convention:
 
-To learn more about Next.js, take a look at the following resources:
-
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
-
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
-
-## Deployment
+```bash
+git checkout -b feature/<feature-name>
+```
 
-The `develop` branch for this repository is automatically deployed to Vercel at [dev.jupiter.utdnebula.com](https://dev.jupiter.utdnebula.com).
+When you are ready to merge your branch into the `develop` branch, please create a pull request and request a review from the Jupiter Dev Team
```

Comment:
Maybe add more details on what should be included in the pull request (issue number, brief description of the changes, etc)